ap_src/ap_app/utils/user_profile.py

```diff
@@ -0,0 +1,15 @@
+# Models
+from ..models import (UserProfile)
+from django.contrib.auth.models import User
+
+def get_user_id_from_username(selected_username):
+    user_matching_username = User.objects.get(username=selected_username)
+    user_id_of_user = int(user_matching_username.id)
+
+    return user_id_of_user
+
+def get_userprofile_id_from_user_id(user_id):
+    userprofile_matching_user_id = UserProfile.objects.get(user_id=user_id)
```

Comment:
Same as above, worth putting any calls to .get() in a try except.

Reply:
Like mentioned earlier, there is a small chance that the user's username is not in the database. This will be addressed with `try` statement.

Reply:
Fixed with ccaa924